### DISCUSSION TITLE
How to connect Planetscale to Grafana

### DISCUSSION THREAD

**Discussion by baptisteArno** [planetscale/discussion#402] [Q&A] [answered]
I'm wondering how I can correctly configure a Planetscale data source on Grafana.

I have selected "MySQL Connection" and filled the info but when trying to log in, it says `Code: UNAVAILABLE\nserver does not allow insecure connections, client must use SSL/TLS"`

But then I don't know what to put in here:

![CleanShot 2023-03-01 at 15 21 29@2x](https://user-images.githubusercontent.com/16015833/222167496-344835a8-63d3-402f-9978-d9d10fbf506e.png)

**Comment by fooforge** [comment, marked as answer]
@baptisteArno Hey 👋,

That should be relatively straight-forward to solve. Uncheck _TLS Client Auth_, check _With CA Cert_ and upload our SSL provider's root certificate. You can download it from https://letsencrypt.org/certs/isrgrootx1.pem.
